app/_data/docs_version_timeline.yml

```diff
@@ -0,0 +1,18 @@
+# Configure timeline of doc articles here
+# Format:
+#   slug: <article's latest slug>
+#   aliases: list of article's renamed slugs (if any)
+#   timeline:
+#     <release_version>: <slug_override>
+#
+# if override value is PAGE_CREATED, all versions below that will be set to redirect to index "/"
+
+-
+  slug: "getting-started/configuring-a-service"
+  aliases: ["getting-started/adding-your-api"]
```

Comment:
Can you explain to me where these `aliases` come from? And why, in this case, is this the alias?

Reply:
The `aliases` are basically names of an article in previous versions. Perhaps I should use a more descriptive name here?

Example:
The **Adding Your API** article (created in 0.2x) was renamed to **Configuring a Service** in 0.13x. The `aliases` property is required to make older versions of the page point to the newer renamed version. For example, when browsing **Adding Your API** in version 0.5x, it would now correctly point to **Configuring a Service** for versions above 0.12x.

Reply:
@vividh What happens/is there a procedure if a page is renamed more than once? Or a page is deleted from a particular version of the docs (e.g. 0.10), and added back in a subsequent version (0.11)?

Reply:
This comment above in the diff seems to answer my question (or the part about multiple renames):

> Note: For a renamed article, specify the latest slug under `slug`, and other slugs under `aliases`.
> Specify the old renamed slug under the latest version that it existed in.

Reply:
> @vividh What happens/is there a procedure if a page is renamed more than once? Or a page is deleted from a particular version of the docs (e.g. 0.10), and added back in a subsequent version (0.11)?

Yes, multiple renames, deletions and creations are handled as well. For example, following is a perfectly valid configuration:
```yml
-
  slug: "getting-started/configuring-a-service"
  aliases: ["getting-started/adding-your-api", "getting-started/service-configuration"]
  timeline:
    0.12.x: "getting-started/service-configuration"
    0.10.x: "PAGE_CREATED"
    0.9.x:  "PAGE_DELETED"
    0.8.x:  "getting-started/adding-your-api"
    0.3.x:  "PAGE_CREATED"
```
and the end result would be:
<img width="431" alt="screen shot 2018-11-10 at 11 05 40 am" src="https://user-images.githubusercontent.com/6534375/48297887-a52c1a80-e4d8-11e8-8dc0-60347323f2fc.png">

Here, versions `0.3.x - 0.8.x` point to `getting-started/adding-your-api`, versions `0.10.x - 0.12.x` point to `getting-started/service-configuration`, and versions above those point to the latest article slug i.e. `getting-started/configuring-a-service`.

Reply:
@vividh Great, thank you. That appeared more and more evident after the comment noted above :) Nice!